### PR TITLE
add some blocks

### PR DESCRIPTION
**Pull request by Bonum-taurus**
https://Bonum-taurus.github.io/Museum/

### CODE REVIEW

**Review by mykhalenych — CHANGES_REQUESTED** [review]
deploy link isn't working

**Comment by Bonum-taurus** [comment]
https://Bonum-taurus.github.io/Museum/

**Comment by Bonum-taurus** [comment]
https://Bonum-taurus.github.io/Museum/

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,367 @@
...
+              <div class="footer__contacts">
+                <a
+                  href="https://goo.gl/maps/8ECKZaozX3ZxoGvw9"
+                  class="footer__info-address"
+                  target="_blank"
+                >
+                  вул. М. Грушевського, 6; Київ, 01001
+                </a>
+
+                <a href="tel: +38 (044) 278-13-57" class="footer__info-tel">
```
Comment (volodymyr-soltys97) [inline src/index.html:321]: Need to fix format phone only in href everywhere
```suggestion
                <a href="tel: +380442781357" class="footer__info-tel">
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Add favicon to page
image

2. Need to remove default styles for input after autocomplete
image

3. Add additional styles for these elements when hovering
image

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- Fix the form. Just clear it after the submission. The page shouldn't reload

https://github.com/mate-academy/Museum/assets/99403198/467f3d90-e03e-4271-9ae6-714786cba281

- Don't hardcode the content width for tablets and mobile. Just use the correct padding 
image

- The image doesn't look good. Fix it 
image

- This image seems too big. Doesn't match the design 
image

**Comment by TarasHoliuk** [comment]
1. The image in the header still might look a bit squeezed in some screen sizes:
image
It might be tricky to fix with the background image, you may try to set it as an img

not critical I guess, but at least try this way :arrow_up: 

2. Let's add a hover effect to images to make site more live

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Looks good!
I've added 2 small comments to improve it a bit - check them

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Still not fixed
image

**Pull request closed without merging**